operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md

```diff
@@ -77,6 +77,16 @@ Use the second person and avoid the first person.
 
 For example: "View the status in the Status pane" as opposed to "We’ll view the status in the Status pane".
 
+An exception is **Customer Education** content which can be written in first or second person to keep the audience focused and interested.
+
+For example: 
+> While Mattermost provides options to make mobile deployment as easy as possible, it will take time and iteration. 
+>
+>We are here to [support](https://link-to-support-section) you.
```

Comment:
```suggestion
**We are here to [support](https://link-to-support-section) you.**
```
Additional suggestion: Add this directly after the preceding sentence (i.e., remove the line space).

Reply:
It is supposed to stand alone to give emphasis to the statement.

And as above why should we not use markdown block quotes when we are quoting?

Reply:
I've provided a comment above. :)